<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>871</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_analogInput">
     <attribute name="title">
      <string>Analog Input</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_start">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>440</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">text-align:left;</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/start.png</normaloff>images/start.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_stop">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>440</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">text-align:left;</string>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/stop.png</normaloff>images/stop.png</iconset>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QFrame" name="frame_parameters">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>321</width>
        <height>401</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Channel Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Input Configuration</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Max Input Limit</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>230</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Min Input Limit</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_rate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Rate (Hz)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_samplesRead">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>310</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Samples to Read</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_channelName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>301</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_mode">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>301</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_inputConfig">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>301</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_maxInput">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">text-align:right;</string>
       </property>
       <property name="markdown">
        <string>10

</string>
       </property>
       <property name="placeholderText">
        <string>10</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_minInput">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>260</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">text-align:right;</string>
       </property>
       <property name="markdown">
        <string>-10

</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_rate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">text-align:right;</string>
       </property>
       <property name="markdown">
        <string>1000

</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_samplesRead">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>340</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">text-align:right;</string>
       </property>
       <property name="markdown">
        <string>1000

</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_plot">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>511</width>
        <height>401</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>471</width>
         <height>291</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="PlotWidget" name="graphWidget" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>471</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Amplitude vs Samples Chart</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_scale">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>20</y>
         <width>141</width>
         <height>26</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Auto-scale chart</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_analogValue">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>350</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="markdown">
        <string>0.00

</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar_finite">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>371</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_analogOutput">
     <attribute name="title">
      <string>Analog Output</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>899</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
